### ISSUE TITLE
Make passwords generic

### ISSUE THREAD

**Issue by runjak** [lingdb/container#6]
While I don't deem it an issue of terrible importance due to the isolation provided by using docker,
it would be nice and certainly good style to have no fixed password in the whole setup.
I shall tackle this issue after #4 and #5.

**Comment by runjak** [comment]
I'll postpone this issue and try to solve #9 first.

**Comment by runjak** [comment]
I've added the `secrets/secrets.sh` script to the repository with a `.gitignore` preventing passwords from a `secrets/secrets` file to be checked in.
The secrets file is created using pwgen if it doesn't exist, and is already integrated into all containers.
I've tested that the setup still works as it used to.
